### ISSUE TITLE
Color not works

### ISSUE THREAD

**Issue by fplini** [WoltersKluwerPL/ng-spin-kit#34]
Hi, if I add [color] attribute at a sk-fadinig-circle I get an error "Can't bind to 'color' since it isn't a known property of 'sk-fading-circle'"

**Comment by gins0023** [comment]
Yeah, it doesn't work on the circle or fading-circle because the styles are restrictive.  Every other component has color attribute..

**Comment by davecoffin** [comment]
So, which does color work on? I just tried on rotating plane and it does not work. I'm also not convinced that your answer is correct @gins0023 because you can manually override the color in chrome dev tools using simple css for fading circle. 
https://cl.ly/jFcW

**Comment by davecoffin** [comment]
Also, this is easily override-able in the module's code, just pull up node_modules > ng2-spin-kit > app > spinner > ANIMATION.ts and edit the inline styles (changing color from #333 to whatever)

**Comment by gins0023** [comment]
I'd have to see your code to see why rotating plane didn't work.  Of course you can modify it in your dev tools, that isn't the problem.  The problem is being able to modify it dynamically through angular.  Specifically, it's with the :before css filter.  I couldn't figure out a way to target that filter dynamically.  You can always target those two components directly with css in your own parent components though..

**Comment by spongessuck** [comment]
@davecoffin Your changes won't be saved if you have to reinstall your modules.
